cityehr-quick-start-guide/pom.xml

```diff
@@ -25,6 +25,13 @@
         <fo.generated-resources>${project.build.directory}/generated-resources/fo</fo.generated-resources>
         <html.generated-resources>${project.build.directory}/generated-resources/html</html.generated-resources>
         <website.output.folder>${project.build.directory}/website</website.output.folder>
+
+        <petal.base-url>https://petal.evolvedbinary.com/</petal.base-url>
+        <petal.github-base-url>https://github.com/</petal.github-base-url>
+        <petal.github-org-name>evolvedbinary</petal.github-org-name>
+        <petal.github-repo-name>cityehr-documentation</petal.github-repo-name>
+        <petal.github-branch>develop</petal.github-branch>
+        <petal.referrer-url>https://evolvedbinary.github.io/cityehr-documentation/</petal.referrer-url>
```

Comment:
Can you rename `petal-referer-url` to `petal.referrer-base-url` please (also in XSLT and CI scripts), and drop the trailing `/` from here (the add it in the concat inside the XSLT instead).

Reply:
Fixed with ad76cc3